src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/LocalAppContextSwitches.netfx.cs

```diff
@@ -7,38 +7,27 @@
 
 namespace Microsoft.Data.SqlClient
 {
-    internal static class LocalAppContextSwitches
+    internal static partial class LocalAppContextSwitches
     {
-        internal const string MakeReadAsyncBlockingString = @"Switch.Microsoft.Data.SqlClient.MakeReadAsyncBlocking";
-        private static int _makeReadAsyncBlocking;
-        public static bool MakeReadAsyncBlocking
-        {
-            [MethodImpl(MethodImplOptions.AggressiveInlining)]
-            get
-            {
-                return LocalAppContext.GetCachedSwitchValue(MakeReadAsyncBlockingString, ref _makeReadAsyncBlocking);
-            }
-        }
-
         internal const string UseMinimumLoginTimeoutString = @"Switch.Microsoft.Data.SqlClient.UseOneSecFloorInTimeoutCalculationDuringLogin";
-        private static int _useMinimumLoginTimeout;
+        private static bool _useMinimumLoginTimeout;
         public static bool UseMinimumLoginTimeout
         {
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get
             {
-                return LocalAppContext.GetCachedSwitchValue(UseMinimumLoginTimeoutString, ref _useMinimumLoginTimeout);
+                return AppContext.TryGetSwitch(UseMinimumLoginTimeoutString, out _useMinimumLoginTimeout) ? _useMinimumLoginTimeout : false;
```

Comment:
return AppContext.TryGetSwitch(UseMinimumLoginTimeoutString, out _useMinimumLoginTimeout) ? _useMinimumLoginTimeout : false;

Reply:
That would change the logic. If TryGetSwitch returns true, it has found the switch and set _useMinimumLoginTimeout to its value and we want to return the value of _useMinimumLoginTimeout. If TryGetSwitch returns false, no setting for the switch was found. In that case, false is the default value we want to use for that switch.